src/main/java/com/surrealdb/driver/model/patch/ReplacePatch.java

```diff
@@ -0,0 +1,19 @@
+package com.surrealdb.driver.model.patch;
```

Comment:
We actually have a custom Patch type `change` which is used for diff-match-patching of text values in a record. It's basically the same as the `replace` Patch, but the `value` is a diff-match-patch string.

Reply:
Just to be clear, this is IN ADDITION to the `replace` patch! So SurrealDB would have:

1. add
2. remove
3. replace
4. change

Reply:
Great, I updated the PR with the changes requested.